test/unit/Escrow.t.sol

```diff
@@ -1034,33 +1195,36 @@ contract EscrowUnitTests is UnitTest {
     // ---
 
     function test_withdrawETH_HappyPath() external {
+        uint256 ethAmount = _stETH.getPooledEthByShares(stethAmount);
         uint256 balanceBefore = _vetoer.balance;
         uint256[] memory unstEthIds = _getUnstEthIdsFromWQ();
 
-        _vetoerLockedStEth(stethAmount);
+        _vetoerLockedStEth(ethAmount);
         _transitToRageQuit();
 
         _withdrawalQueue.setRequestWithdrawalsResult(unstEthIds);
 
-        _ensureUnstEthAddedToWithdrawalsBatchesQueue(unstEthIds, stethAmount);
-        _claimStEthViaWQ(unstEthIds, stethAmount);
+        _ensureUnstEthAddedToWithdrawalsBatchesQueue(unstEthIds);
+        _claimStEthViaWQ(unstEthIds, ethAmount);
         _ensureRageQuitExtensionPeriodStartedNow();
         assertTrue(_escrow.getRageQuitEscrowDetails().isRageQuitExtensionPeriodStarted);
 
         _wait(Durations.from(1));
 
         vm.startPrank(_vetoer);
         vm.expectEmit();
-        emit AssetsAccounting.ETHWithdrawn(_vetoer, SharesValues.from(stethAmount), ETHValues.from(stethAmount));
+        emit AssetsAccounting.ETHWithdrawn(
+            _vetoer, SharesValues.from(_stETH.getSharesByPooledEth(ethAmount)), ETHValues.from(ethAmount)
+        );
         _escrow.withdrawETH();
         vm.stopPrank();
 
-        assertEq(_vetoer.balance, balanceBefore + stethAmount);
+        assertEq(_vetoer.balance, balanceBefore + ethAmount);
 
         IEscrow.SignallingEscrowDetails memory signallingEscrowDetails = _escrow.getSignallingEscrowDetails();
 
-        assertEq(signallingEscrowDetails.totalStETHLockedShares.toUint256(), stethAmount);
-        assertEq(signallingEscrowDetails.totalStETHClaimedETH.toUint256(), stethAmount);
+        assertEq(signallingEscrowDetails.totalStETHLockedShares.toUint256(), _stETH.getSharesByPooledEth(ethAmount));
+        assertEq(signallingEscrowDetails.totalStETHClaimedETH.toUint256(), ethAmount);
```

Comment:
Also, it would be nice to check that after the withdrawal of ETH, other attempts of the vetoer to withdraw ETH will fail